ff/serving/provider/online.go

```diff
@@ -164,6 +239,85 @@ func (store *redisOnlineStore) CreateTable(feature, variant string, valueType Va
 	}
 	table := &redisOnlineTable{client: store.client, key: key, valueType: valueType}
 	return table, nil
+
+}
+
+// CASSANDRA CREATE TABLE
+func (store *cassandraOnlineStore) CreateTable(feature, variant string, valueType ValueType) (OnlineStoreTable, error) {
+
+	//Create table Name
+	tableName := fmt.Sprintf("%s.table%s", store.keyspace, sn.Custom(feature, "[^a-zA-Z0-9_]"))
+
+	//Create table key
+	key := cassandraTableKey{store.keyspace, feature, variant}
+	keyName := sn.Custom(key.String(), "[^a-zA-Z0-9_]")
+
+	//Check if table exists
+	getTable, _ := store.GetTable(feature, variant)
+	if getTable != nil {
+		return nil, &TableAlreadyExists{feature, variant}
+	}
+
+	// Create Table
+	query := fmt.Sprintf("CREATE TABLE %s (entity text PRIMARY KEY, value text)", tableName)
+	err := store.session.Query(query).WithContext(ctx).Exec()
+	if err != nil {
+		return nil, err
+	}
+
+	//return the table
+	table := &cassandraOnlineTable{
+		cluster:   store.cluster,
+		session:   store.session,
+		key:       key,
+		valueType: valueType,
+	}
+
+	//Update Metadata
+	metadataTableName := fmt.Sprintf("%s.tableMetadata", store.keyspace)
```

Comment:
Let's add to the metadata before creating the table, I think its safer in case of failure